tests/unit/lang/translator/CMakeLists.txt

```diff
@@ -15,8 +15,11 @@ add_executable(
   ${CMAKE_SOURCE_DIR}/src/lang/translation_provider_CPUFLASH.cpp
   ${CMAKE_SOURCE_DIR}/src/lang/unaccent.cpp
   ${CMAKE_SOURCE_DIR}/tests/unit/test_main.cpp
+  ${CMAKE_SOURCE_DIR}/src/lang/gettext_string_hash.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/providerCPUFLASH.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/hash.cpp
+  # we need to add mo file generation to test files
```

Comment:
no, just use one MO file from our translations and put it into the unit test - that's good enough for the prototype implementation